src/components/ButtonPanel.js

```diff
@@ -1,38 +1,47 @@
 import React from 'react';
+import PropTypes from 'prop-types';
 import Button from './Button';
 
-const ButtonPanel = () => (
-  <div className="panel">
-    <div className="group-button">
-      <Button name="AC" color="#dfe4ea" />
-      <Button name="+/-" color="#dfe4ea" />
-      <Button name="%" color="#dfe4ea" />
-      <Button name="÷" />
-    </div>
-    <div className="group-button">
-      <Button name="7" color="#dfe4ea" />
-      <Button name="8" color="#dfe4ea" />
-      <Button name="9" color="#dfe4ea" />
-      <Button name="X" />
-    </div>
-    <div className="group-button">
-      <Button name="4" color="#dfe4ea" />
-      <Button name="5" color="#dfe4ea" />
-      <Button name="6" color="#dfe4ea" />
-      <Button name="-" />
-    </div>
-    <div className="group-button">
-      <Button name="1" color="#dfe4ea" />
-      <Button name="2" color="#dfe4ea" />
-      <Button name="3" color="#dfe4ea" />
-      <Button name="+" />
-    </div>
-    <div className="group-button">
-      <Button name="0" wide color="#dfe4ea" />
-      <Button name="." color="#dfe4ea" />
-      <Button name="=" />
+const ButtonPanel = props => {
+  const { clickHandler } = props;
+
+  return (
+    <div className="panel">
+      <div className="group-button">
+        <Button name="AC" color="#dfe4ea" clickHandler={clickHandler} />
+        <Button name="+/-" color="#dfe4ea" clickHandler={clickHandler} />
+        <Button name="%" color="#dfe4ea" clickHandler={clickHandler} />
+        <Button name="÷" clickHandler={clickHandler} />
+      </div>
+      <div className="group-button">
+        <Button name="7" color="#dfe4ea" clickHandler={clickHandler} />
+        <Button name="8" color="#dfe4ea" clickHandler={clickHandler} />
+        <Button name="9" color="#dfe4ea" clickHandler={clickHandler} />
+        <Button name="X" clickHandler={clickHandler} />
+      </div>
+      <div className="group-button">
+        <Button name="4" color="#dfe4ea" clickHandler={clickHandler} />
+        <Button name="5" color="#dfe4ea" clickHandler={clickHandler} />
+        <Button name="6" color="#dfe4ea" clickHandler={clickHandler} />
+        <Button name="-" clickHandler={clickHandler} />
+      </div>
+      <div className="group-button">
+        <Button name="1" color="#dfe4ea" clickHandler={clickHandler} />
+        <Button name="2" color="#dfe4ea" clickHandler={clickHandler} />
+        <Button name="3" color="#dfe4ea" clickHandler={clickHandler} />
+        <Button name="+" clickHandler={clickHandler} />
+      </div>
+      <div className="group-button">
+        <Button name="0" wide color="#dfe4ea" clickHandler={clickHandler} />
+        <Button name="." color="#dfe4ea" clickHandler={clickHandler} />
+        <Button name="=" clickHandler={clickHandler} />
+      </div>
```

Comment:
**ButtonPanel component**
- It implements `handleClick` function  [screenshot](https://gitlab.com/microverse/guides/projects/requirements_screenshots/raw/master/images/react_reducx/calculator/milestone5/panel/implementHandleClick.png) then `handleClick` should be passed as `clickHandler` props to each `button` [screenshot](https://gitlab.com/microverse/guides/projects/requirements_screenshots/raw/master/images/react_reducx/calculator/milestone5/panel/passToButton.png)